torchchat/model.py

```diff
@@ -11,6 +11,7 @@
 from enum import Enum
 from pathlib import Path
 from typing import Callable, Dict, Optional, Union
+from abc import ABC, abstractmethod
```

Comment:
ABC is unused?

Reply:
should be one of the Model's parents. Fixed it.